lib/httpHandler.ts

```diff
@@ -0,0 +1,53 @@
+import { EventWithMiddlewareContext } from "somod";
+import {
+  Copy,
+  CustomEventType,
+  HttpResponse,
+  LambdaFunctionType
+} from "./types";
+
+export class HttpLambda {
+  private apis: Record<string, LambdaFunctionType> = {};
+
+  register = <TBody = Record<string, unknown>>(
+    routeKey: string,
```

Comment:
instead of `routeKey` , lets have `route`, `method`

Reply:
done